app/src/helper/arweave.ts

```diff
@@ -0,0 +1,46 @@
+import Arweave from 'arweave/web';
+import {JWKInterface} from "arweave/web/lib/wallet";
+
+export const ARWEAVE_URL_START = "https://arweave.net/";
+
+const arweave = Arweave.init({});
+arweave.network.getInfo().then(console.log);
+
+const logBalance = (key: JWKInterface) => {
+  return arweave.wallets.jwkToAddress(key).then((address) => {
+    arweave.wallets.getBalance(address).then((balance) => {
+      console.log(arweave.ar.winstonToAr(balance));
+    });
+  });
+};
+
+
+const logTransactionData = async (id: string) => {
+  arweave.transactions.getData(id, {decode: true, string: true}).then(data => {
+    console.log(data);
+  });
+};
+
+export const saveDocumentToArweave = async (file: string): Promise<string> => {
+  // TODO: get user's key somehow
```

Comment:
Yeah. That was my concern - can we add a hardcoded one yet?

Reply:
You can request a wallet that comes with 1AR which is what I've done so far, not sure if you can link between eth wallets or any other useful stuff or whether it's literally just the key for an Arweave specific thing. Are there any support people from Arweave we can talk to through FoB?

Reply:
There are - I'll send a message about this through to them